pkg/cmd/roachtest/test_runner.go

```diff
@@ -1171,8 +1171,8 @@ func PredecessorVersion(buildVersion version.Version) (string, error) {
 	// (see runVersionUpgrade). The same is true for adding a new key to this
 	// map.
```

Comment:
The fixtures also need to be updated like in [this PR](https://github.com/cockroachdb/cockroach/pull/50770).

Note, there were some issues with this in the last couple releases that should be fixed now. If you need it, see https://github.com/cockroachdb/cockroach/pull/55253 for reference.